training_config/biattentive_classification_network.json

```diff
@@ -0,0 +1,72 @@
+{
+  "dataset_reader":{
+    "type": "sst_tokens",
+    "use_subtrees": true,
+    "granularity": "5-class"
+  },
+  "validation_dataset_reader":{
+    "type": "sst_tokens",
+    "use_subtrees": false,
+    "granularity": "5-class"
+  },
+  "train_data_path": "https://homes.cs.washington.edu/~nfliu/files/sst/train.txt",
+  "validation_data_path": "https://homes.cs.washington.edu/~nfliu/files/sst/dev.txt",
+  "test_data_path": "https://homes.cs.washington.edu/~nfliu/files/sst/test.txt",
+  "model": {
+    "type": "bcn",
+    "text_field_embedder": {
+      "tokens": {
+        "pretrained_file": "https://homes.cs.washington.edu/~nfliu/files/misc/glove.840B.300d.txt.gz",
```

Comment:
Similarly here, these are on AI2's s3 bucket.  No need to change it if you don't want to; up to you.

Reply:
Can you link me to this? The link I would have guessed it would be at (https://s3-us-west-2.amazonaws.com/allennlp/datasets/glove/glove.840B.300d.txt.gz) seems to return a 404. Note that 840B, not 6B.

Reply:
Hmm, looks like you're right, it was missing.  I just added it.